commands/ls/ls.go

```diff
@@ -0,0 +1,52 @@
+package ls
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	CmdLs = "ls"
+)
+
+
+func Command() *cli.Command {
+	return &cli.Command{
+		Name:        CmdLs,
+		HelpName:    CmdLs,
+		Action:      Action,
+		ArgsUsage:   ` `,
+		Usage:       `The ls utility lists directory contents of files and directories`,
+		Description: `The ls utility lists directory contents of files and directories`,
+		Flags:       Flags(),
+	}
+}
+
+func Flags() []cli.Flag {
+	return []cli.Flag{
+		// TODO: Add flags
+	}
+}
+
+func Action(c *cli.Context) error {
+	args := c.Args()
+	var files,err = ioutil.ReadDir(args.First())
+
+	if args.Len() == 0 {
+		files,err = ioutil.ReadDir("./")
+	}
+
+	if err != nil {
+		log.Println(err)
+		return err
+	}
+
+	for _, f := range files {
+		fmt.Println(f.Name())
+	}
+
+	return nil
+}
```

Comment:
Can we refactor this something like this:

```suggestion
func Action(c *cli.Context) error {
	path := c.Args().First()
	if path == "" {
		path = "./"
	}
 	files, err := os.ReadDir(path)
 	if err != nil {
 		return fmt.Errorf("read dir %q error: %w", path, err)
 	}
 	for _, f := range files {
 		fmt.Println(f.Name())
 	}

 	return nil
 }
```

Reply:
I wrote a bit of javascript style as a frontend developer 😄. I will update 👍